runtime/helpers/elementHelpers.js

```diff
@@ -1,159 +1,179 @@
+			const script = `document.querySelectorAll('${selectors[i]}').forEach(element => element.style.opacity = '0')`;
 
 			await driver.execute(script);
-		}},
+		}
+	},
+
+	hideElements: async function (selectors) {
+		// if arg is no array make it one
+		selectors = typeof selectors == "string" ? [selectors] : selectors;
+		for (let i = 0; i < selectors.length; i++) {
+			const script = `document.querySelectorAll('${selectors[i]}').forEach(element => element.style.opacity = '0')`;
 
-        hideElements: async function(selectors) {
-            // if arg is no array make it one
-            selectors = typeof selectors == 'string' ? [selectors] : selectors;
-            for (let i = 0; i < selectors.length; i++) {
-                const script = `document.querySelectorAll('${
-                    selectors[i]
-                }').forEach(element => element.style.opacity = '0')`;
-    
-                await driver.execute(script);
-            }
-        },
-        /**
-            * showElements show elements
-            * @param  string  selectors   css selector or array of css selectors
-            */
-        showElements: async function(selectors) {
-            // if arg is no array make it one
-            selectors = typeof selectors == 'string' ? [selectors] : selectors;
-            for (let i = 0; i < selectors.length; i++) {
-                const script = `document.querySelectorAll('${
-                    selectors[i]
-                }').forEach(element => element.style.opacity = '1')`;
-    
-                await driver.execute(script);
-            }
-        },
-        /**
-		* clicks an element (or multiple if present) that is not visible,
-		* useful in situations where a menu needs a hover before a child link appears
-		* @param {string} css selector used to locate the elements
-		* @param {string} text to match inner content (if present)
-		* @example
-		*    this.clickHiddenElement('nav[role="navigation"] ul li a','School Shoes');
-		*/
+			await driver.execute(script);
+		}
+	},
+	/**
+	 * showElements show elements
+	 * @param  string  selectors   css selector or array of css selectors
+	 */
+	showElements: async function (selectors) {
+		// if arg is no array make it one
+		selectors = typeof selectors == "string" ? [selectors] : selectors;
+		for (let i = 0; i < selectors.length; i++) {
+			const script = `document.querySelectorAll('${selectors[i]}').forEach(element => element.style.opacity = '1')`;
+
+			await driver.execute(script);
+		}
+	},
+	/**
+	 * clicks an element (or multiple if present) that is not visible,
+	 * useful in situations where a menu needs a hover before a child link appears
+	 * @param {string} css selector used to locate the elements
+	 * @param {string} text to match inner content (if present)
+	 * @example
+	 *    this.clickHiddenElement('nav[role="navigation"] ul li a','School Shoes');
+	 */
 	clickHiddenElement: function (cssSelector, textToMatch) {
```

Comment:
I know you didn't change it. Do we have a function inside a function? Let's talk about that, please.